### PR TITLE
11.1.0 Changelog

### PR DESCRIPTION
**Pull request by ryanthemanuel**

### CODE REVIEW

**Review by flotwig — APPROVED** [review]

File: content/_changelogs/11.1.0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  [#23911](https://github.com/cypress-io/cypress/issues/23911)
+- Sessions will no longer be stored if validation fails. Fixes
+  [#24208](https://github.com/cypress-io/cypress/issues/24208)
+- Updates were made to enhance the cy.session() recovered error experience. With
+  this change, cy.session() will no longer fail validation when false is
+  returned from the validate callback. Fixes
+  [#21398](https://github.com/cypress-io/cypress/issues/21398),
+  [#24205](https://github.com/cypress-io/cypress/issues/24205), and
+  [#24593](https://github.com/cypress-io/cypress/issues/24593).
+- `mount()` now accept a extensions.directives option, which will be registered
```
Comment (mschile) [inline content/_changelogs/11.1.0.md:28]:
```suggestion
- `mount()` now accepts an `extensions.directives` option, which will be registered
```

File: content/_changelogs/11.1.0.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+_Released 11/14/2022_
+
+**Features:**
+
+- Next.js v13 is now supported.
+  [#24396](https://github.com/cypress-io/cypress/pull/24396)
+
+**Bugfixes:**
+
+- Fixed an 11.0.0 regression where users couldn't migrate to 11 without going
```
Comment (marktnoonan) [inline content/_changelogs/11.1.0.md:12]:
```suggestion
- Fixed an 11.0.0 regression where Typescript projects couldn't migrate to 11 without going
```

File: content/_changelogs/11.1.0.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Fixed Typescript types for `testIsolation`. Fixes
+  [#23911](https://github.com/cypress-io/cypress/issues/23911)
+- Sessions will no longer be stored if validation fails. Fixes
+  [#24208](https://github.com/cypress-io/cypress/issues/24208)
+- Updates were made to enhance the cy.session() recovered error experience. With
+  this change, cy.session() will no longer fail validation when false is
+  returned from the validate callback. Fixes
+  [#21398](https://github.com/cypress-io/cypress/issues/21398),
+  [#24205](https://github.com/cypress-io/cypress/issues/24205), and
+  [#24593](https://github.com/cypress-io/cypress/issues/24593).
```
Comment (emilyrohrbough) [inline content/_changelogs/11.1.0.md:27]: We had chatted about this on slack. This is more of an enhancement vs bug fix, but this should prob be called out under `Experimental Breaking Changes:` with the change to return false

File: content/_changelogs/11.1.0.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- `mount()` now accepts an `extensions.directives` option, which will be registered
```
Comment (emilyrohrbough) [inline content/_changelogs/11.1.0.md:28]: This sounds like a feature. If this is a bug fix, what does this resolve?

File: content/_changelogs/11.1.0.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+- Fixed an 11.0.0 regression where Typescript projects couldn't migrate to 11 without going
+  through 10 first. Fixes
+  [#24643](https://github.com/cypress-io/cypress/issues/24643)
+- Fixed an 11.0.0 regression where `cypress run` crashes when using the junit
+  reporter and an assertion fails. Fixes
+  [#24652](https://github.com/cypress-io/cypress/issues/24652)
+- Fixed Typescript types for `testIsolation`. Fixes
+  [#23911](https://github.com/cypress-io/cypress/issues/23911)
+- Sessions will no longer be stored if validation fails. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/11.1.0.md:20]: This prevents a previously failed session from being restored in the next test.

File: content/_changelogs/11.1.0.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+**Features:**
+
+- Next.js v13 is now supported.
+  [#24396](https://github.com/cypress-io/cypress/pull/24396)
+
+**Bugfixes:**
+
+- Fixed an 11.0.0 regression where Typescript projects couldn't migrate to 11 without going
+  through 10 first. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/11.1.0.md:13]: Won't all project always need to be migrated to 10 before the 11.0 migration occurs?
```suggestion
- Fixed an 11.0.0 regression where the Migration flow would error and hang for Typescript projects. Fixes
```

File: content/_changelogs/11.1.0.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  [#24643](https://github.com/cypress-io/cypress/issues/24643)
+- Fixed an 11.0.0 regression where `cypress run` crashes when using the junit
```
Comment (emilyrohrbough) [inline content/_changelogs/11.1.0.md:15]:
```suggestion
- Fixed an 11.0.0 regression where `cypress run` crashed when using the junit
```

File: content/_changelogs/11.1.0.md (hunk 6 above)
Comment (ryanthemanuel) [reply content/_changelogs/11.1.0.md:13]: Not if they're already on 10? But I can probably clarify this.

File: content/_changelogs/11.1.0.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  globally for Vue2 component testing.
+  [#24488](https://github.com/cypress-io/cypress/pull/24488)
+
+**Bugfixes:**
+
+- Fixed an 11.0.0 regression where the migration workflow would error and hang
+  for TypeScript projects. Fixes
+  [#24643](https://github.com/cypress-io/cypress/issues/24643)
+- Fixed an 11.0.0 regression where `cypress run` crashed when using the junit
+  reporter and an assertion fails. Fixes
```
Comment (ryanthemanuel) [inline content/_changelogs/11.1.0.md:19]:
```suggestion
  reporter and an assertion failed. Fixes
```

File: content/_changelogs/11.1.0.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+**Bugfixes:**
+
+- Fixed an 11.0.0 regression where the migration workflow would error and hang
+  for TypeScript projects. Fixes
+  [#24643](https://github.com/cypress-io/cypress/issues/24643)
+- Fixed an 11.0.0 regression where `cypress run` crashed when using the junit
+  reporter and an assertion failed. Fixes
+  [#24652](https://github.com/cypress-io/cypress/issues/24652)
+- Fixed Typescript types for `testIsolation`. Fixes
```
Comment (ryanthemanuel) [inline content/_changelogs/11.1.0.md:21]:
```suggestion
- Fixed TypeScript types for `testIsolation`. Fixes
```

File: content/_changelogs/11.1.0.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+- Fixed an 11.0.0 regression where the migration workflow would error and hang
+  for TypeScript projects. Fixes
+  [#24643](https://github.com/cypress-io/cypress/issues/24643)
+- Fixed an 11.0.0 regression where `cypress run` crashed when using the junit
+  reporter and an assertion failed. Fixes
+  [#24652](https://github.com/cypress-io/cypress/issues/24652)
+- Fixed TypeScript types for `testIsolation`. Fixes
+  [#23911](https://github.com/cypress-io/cypress/issues/23911)
+- Sessions will no longer be stored if validation fails preventing a previously
```
Comment (ryanthemanuel) [inline content/_changelogs/11.1.0.md:23]:
```suggestion
- Sessions will no longer be stored if validation fails which will prevent a previously
```

File: content/_changelogs/11.1.0.md (hunk 6 above)
Comment (marktnoonan) [reply content/_changelogs/11.1.0.md]: Cypress 11 migration is the same as Cypress 10 migration, it's based on whether we detect a `cypress.json` file. People are able to migrate directly to 11 from 9 or lower.

**Review by emilyrohrbough — APPROVED** [review]

**Pull request merged**